test/suite/run.py

```diff
@@ -318,6 +321,21 @@ def error(exitval, prefix, msg):
             if option == '-asan':
                 asan = True
                 continue
+            if option == '-batch' or option == 'b':
+                if batchtotal != 0 or len(args) == 0:
+                    usage()
+                    sys.exit(2)
+                b = args.pop(0).split('/')
+                try:
+                    batchnum = int(b[0])
+                    batchtotal = int(b[1])
+                except:
+                    usage()
+                    sys.exit(2)
```

Comment:
Will using multiple assignments for line 328 e.g. `batchnum, batchtotal = args.pop(0).split('/')` make the code more readable? The int casting may make it over-complicated though. 
What sort of scenario the except aims to capture?

Reply:
The except catches a missing '/' (since b won't be a list), it also catches if either "side" is not an integer, like `-b x/12`.  I like your readability suggestion, if I put the assignment in the try block it will catch the missing '/' there.

Reply:
I'll also add comments so it's clearer.